src/main/resources/templates/dev-hub/authentication.html

```diff
@@ -10,57 +10,48 @@
 <body>
 
 <div layout:fragment="content">
-  <h1 class="govuk-heading-xl">Authentication</h1>
+  <h1 class="govuk-heading-xl">How to add an API key</h1>
   <h2 class="govuk-heading-l">API key authentication</h2>
 
-  <p class="govuk-body">The Companies House API requires
-    authentication credentials, in the form of an API key, to be
-    sent with each request.</p>
-
-  <p class="govuk-body">
-    To obtain an API key, go to <a href=https://www.example.com
-                                   class="govuk-link">Your applications</a> and register
-    the application with the Companies House Developer Hub as an
-    <code>API Key</code>
-    application. This will allocate a unique key to the
-    application which can be sent with any
-    <code>GET</code>
-    request for a public resource served by the Companies House
-    API.
+  <p class="govuk-body">The Companies House API requires authentication credentials to be sent with
+    each request, which take the form of an API key.</p>
+  <p class="govuk-body">To add an API key:
   </p>
+  <ol class="list list-number govuk-body">
+    <li>Go to <a href="https://www.example.com/">Application overview</a></li>
+    <li>Select the application you’d like to add a key to, or select Add an application to create a
+      new one.
+    </li>
+    <li>Select Add new key at the bottom of the page.</li>
+    <li>Provide a name for the key and choose the client type.</li>
+    <li>Select Create.</li>
+    <li>A unique key will be generated, which can be sent with any GET request for a public resource
+      served by the Companies House API.
+    </li>
+  </ol>
+
 
   <h3 class="govuk-heading-m">Sending the API key</h3>
   <p class="govuk-body">
-    The Companies House API uses <a
-      href="http://en.wikipedia.org/wiki/Basic_access_authentication" class="govuk-link">HTTP
-    Basic Authentication</a> to transmit an API key between the
-    client application and the server. Basic authentication is
-    usually made up of a username and password pair; the
-    Companies House API takes the username as the API key and
-    ignores the password, so can be left blank.
+    The Companies House API uses HTTP basic access authentication to send an API key between the
+    client application and the server.
+
+
```

Comment:
Remove redundant blank lines - they will be collapsed by HTML to a space.

Should these be separate `<p>` paragraph tags?

Reply:
Sure